headers/linux/netfilter.h

```diff
@@ -0,0 +1,114 @@
+#ifndef _LINUX_NETFILTER_H
+#define _LINUX_NETFILTER_H
+
+#include <stdbool.h>
+#include <linux/types.h>
+#include <bpf/bpf_helpers.h>
+#include <xdp/parsing_helpers.h>
+
+#include "hlist.h"
+
+struct flow_ports {
+	__be16 source, dest;
+} __attribute__((preserve_access_index));
```

Comment:
Why the attribute?

Reply:
@tohojo I just copied it from kernel source code. I guess we can remove it